library/std/src/os/unix/net/ancillary.rs

```diff
@@ -326,11 +417,15 @@ pub struct ScmCredentials<'a>(AncillaryDataIter<'a, ()>);
 #[unstable(feature = "unix_socket_ancillary_data", issue = "76915")]
 pub struct ScmCredentials<'a>(AncillaryDataIter<'a, libc::ucred>);
 
+#[cfg(freebsd13)]
```

Comment:
You need to change the `#[cfg(` statement on line 409 or else this will be defined twice.